dot/parachain/availability-store/availability_store.go

```diff
@@ -79,12 +85,38 @@ type AvailabilityStoreSubsystem struct {
 	SubSystemToOverseer chan<- any
 	OverseerToSubSystem <-chan any
 	availabilityStore   availabilityStore
-	pruningConfig       pruningConfig
+	currentMessage      any
+	finalizedNumber     parachaintypes.BlockNumber
+	knownBlocks         KnownUnfinalizedBlock
+	pruningConfig       PruningConfig
 	clock               subsystemClock
```

Comment:
not sure what the point of this attribute is given it's not stateful.  Couldn't you just reference a helper function to give you the `BETimestamp` of `time.Now()`?

Reply:
Good point, I've updated this.